app/src/main/AndroidManifest.xml

```diff
@@ -497,7 +498,20 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="de.tum.in.tumcampusapp.activities.MainActivity" />
         </activity>
-
+        <activity
+            android:name=".activities.BarrierFreeInfoActivity"
+            android:configChanges="orientation|screenSize"
+            android:label="@string/barrier_free"
+            android:launchMode="singleTop"
+            android:parentActivityName=".activities.MainActivity"></activity>
+        <activity
+            android:name=".activities.BarrierFreeContactActivity"
+            android:configChanges="orientation|screenSize"
+            android:label="@string/contact_person"
+            android:launchMode="singleTop"
+            android:parentActivityName=".activities.BarrierFreeInfoActivity"
+            >
+        </activity>
```

Comment:
better don't use "code cleanup" on code that is not yours (if you want you can let the ide cleanup a selection only)